test/types/string/sungeun/c_string/casts.chpl

```diff
@@ -48,5 +48,5 @@ writeln(n:string);
 writeln(r:string);
 writeln(i:string);
 writeln(c:string);
-writeln(e:string);
+writeln(pe:string);
```

Comment:
This one similarly just makes me go "ugh" and also "Could we quickly start moving some things into Math vs. AutoMath before deciding where absolutely everything lives?"